cms/search/views.py

```diff
@@ -25,11 +25,12 @@ def search(request):
     date_to=2020-11-29
     """
 
-    query_string = request.GET.get("query", None)
+    query_string = request.GET.get("query", "")
     search_ordering = request.GET.get("order", "-latest_revision_created_at")
     search_type = request.GET.get("content_type", "")
     date_from = request.GET.get("date_from", "")
     date_to = request.GET.get("date_to", "")
+    content_type = request.GET.get("content_type", "")
```

Comment:
This looks suspiciously close to line :30, is it right?

Reply:
It isn't. I was getting confused by the different names for the same thing.